core/src/actors/chain_manager/mod.rs

```diff
@@ -364,11 +419,16 @@ impl ChainManager {
         Ok(missing_inv_entries)
     }
 
+    fn get_block_to_validate(&mut self, hash: &Hash) -> Option<Block> {
```

Comment:
I would call this function `remove_block_to_validate` instead, `get_` kind of implies it won't modify anything which is in contradiction with the `&mut self` demanded by the method